pages/_app.tsx

```diff
@@ -1,8 +1,28 @@
-import '../styles/globals.css';
+import * as React from 'react';
```

Comment:
Try importing the only thing you want... Don't import everything.

you may check [this](https://stackoverflow.com/questions/54585763/what-is-the-difference-between-import-as-react-from-react-vs-import-react-fr)